.github/workflows/update-website.yaml

```diff
@@ -0,0 +1,19 @@
+name: Trigger Website Update
+on:
+  push:
+    paths:
+    - 'docs/**'
+    branches:
+    - main
+permissions:
+  contents: read
+jobs:
+  trigger_website_update:
+    name: Trigger Website Update
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger via gh
+        env:
+          GH_TOKEN: ${{ secrets.IG_WEBSITE_TOKEN }}
+        run: |
+           gh workflow run -R inspektor-gadget/website azure-static-web-apps-polite-mushroom-00d947803.yml
```

Comment:
This one is to update the "latest" docs, we still need something else to handle the versions. Probably when a release is done, we could trigger the job in https://github.com/inspektor-gadget/website/blob/main/.github/workflows/update-inspektor-gadget-release.yaml instead of running it daily. But I'll leave this change for another PR.